frontend/utils/constants.js

```diff
@@ -20,6 +20,7 @@ export const CHAT_ATTACHMENT_SUPPORTED_EXTENSIONS = [
 // TODO: fetch this value from a server API
 export const MEGABYTE = 1 << 20
 export const CHAT_ATTACHMENT_SIZE_LIMIT = 30 * MEGABYTE // in byte.
+export const IMAGE_ATTACHMENT_MAX_SIZE = 400000 // 400KB
```

Comment:
Can you create a new entry for `KILOBYTE` just below `MEGABYTE`?

```js
export const KILOBYTE = 1 << 10
```

Then:

```js
export const IMAGE_ATTACHMENT_MAX_SIZE = 400 * KILOBYTE
```

This is a slightly larger number than 400000

Reply:
Updated.